ssz-rs/src/container.rs

```diff
@@ -1,6 +1,6 @@
 #[cfg(test)]
 mod tests {
-    use crate::prelude::*;
+    use crate::{lib::*, prelude::*};
```

Comment:
why do we need to pull in the `lib` imports into the tests now?

Reply:
oh actually, we don't necessarily need to run the tests in no_std, so we don't need this as well. removing